@types/model/V5ClusterDetailsResponse.d.ts

```diff
@@ -69,4 +69,8 @@ export default class V5ClusterDetailsResponse {
     * @member {Array.<module:model/V5ClusterDetailsResponseVersions>} versions
     */
     versions: V5ClusterDetailsResponseVersions[];
+    /**
+    * @member {module:model/V5ClusterLabels} labels
+    */
+    labels: {[labelKey: string]: string};
```

Comment:
@axbarsan this is not right, right?

Reply:
but why not the `V5ClusterLabels` type?

Reply:
Better import `V5ClusterLabels` and reference it?

Reply:
yasss